mibitracker/request_helpers.py

```diff
@@ -612,21 +614,31 @@ def _check_status(response):
         return response
 
     def get(self, *args, **kwargs):
+        if 'timeout' not in kwargs:
+            kwargs.update({'timeout': 10})
```

Comment:
I know it's only two lines but a simple helper function for this would be nice. Good call checking first so that the user can specify their own timeout if they want.

Reply:
done